ibis/backends/flink/tests/test_ddl.py

```diff
@@ -28,36 +56,12 @@ def reset_con(con):
 
 @pytest.fixture
 def awards_players_schema():
-    return sch.Schema(
-        {
-            "playerID": dt.string,
-            "awardID": dt.string,
-            "yearID": dt.int32,
-            "lgID": dt.string,
-            "tie": dt.string,
-            "notes": dt.string,
-        }
-    )
+    return _awards_players_schema
 
 
 @pytest.fixture
 def functiona_alltypes_schema():
-    return sch.Schema(
-        {
-            "id": dt.int32,
-            "bool_col": dt.bool,
-            "smallint_col": dt.int16,
-            "int_col": dt.int32,
-            "bigint_col": dt.int64,
-            "float_col": dt.float32,
-            "double_col": dt.float64,
-            "date_string_col": dt.string,
-            "string_col": dt.string,
-            "timestamp_col": dt.timestamp(scale=3),
-            "year": dt.int32,
-            "month": dt.int32,
-        }
-    )
+    return _functiona_alltypes_schema
```

Comment:
Nit: Why is it `_functiona` instead of `_functional`? Is it just a typo that is getting propagated?

Reply:
not sure, i did not change the previous spelling in this PR. @chloeh13q can you help confirming the intent?

Reply:
Not sure, I didn't change the previous implementation. @chloeh13q can you help confirming the intent?

Reply:
Almost certainly just a typo, since I don't see this anywhere else. Doesn't matter to me, can leave it and I'll fix it in a quick PR later, or can update it here. 🤷

Reply:
let's fix in a separate pr